.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Great work adding test.md to your .gitignore file, please also add the `node_modules` directory as it can be added by accident while setting up linters. :+1:

Reply:
ignore node_modules directory done!